app/assets/stylesheets/application.scss

```diff
@@ -24,6 +24,10 @@ body {
   background-size: cover;
 }
 
+#user_profile {
```

Comment:
Avoid using id selectors<br>Selector `user_profile` should be written in lowercase with hyphens